src/Symfony/Component/Serializer/InvariantViolation.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer;
+
+class InvariantViolation
+{
+    /**
+     * @var mixed
+     */
+    private $normalizedValue;
+
+    /**
+     * @var string
+     */
+    private $message;
+
+    /**
+     * @var \Throwable|null
+     */
```

Comment:
Same on these 3 props - phpdoc not necessary because of the constructor docs/types

Reply:
Removed.